openslides/motions/signals.py

```diff
@@ -57,31 +58,36 @@ def create_builtin_workflows(sender, **kwargs):
                                      action_word='Accept',
                                      recommendation_label='Acceptance',
                                      css_class='success',
-                                     merge_amendment_into_final=True)
+                                     merge_amendment_into_final=1)
     state_2_4 = State.objects.create(name=ugettext_noop('rejected'),
                                      workflow=workflow_2,
                                      action_word='Reject',
                                      recommendation_label='Rejection',
-                                     css_class='danger')
+                                     css_class='danger',
+                                     merge_amendment_into_final=-1)
     state_2_5 = State.objects.create(name=ugettext_noop('withdrawed'),
                                      workflow=workflow_2,
                                      action_word='Withdraw',
-                                     css_class='default')
+                                     css_class='default',
+                                     merge_amendment_into_final=-1)
     state_2_6 = State.objects.create(name=ugettext_noop('adjourned'),
                                      workflow=workflow_2,
                                      action_word='Adjourn',
                                      recommendation_label='Adjournment',
-                                     css_class='default')
+                                     css_class='default',
+                                     merge_amendment_into_final=-1)
     state_2_7 = State.objects.create(name=ugettext_noop('not concerned'),
                                      workflow=workflow_2,
                                      action_word='Do not concern',
                                      recommendation_label='No concernment',
-                                     css_class='default')
+                                     css_class='default',
+                                     merge_amendment_into_final=-1)
     state_2_8 = State.objects.create(name=ugettext_noop('refered to committee'),
                                      workflow=workflow_2,
                                      action_word='Refer to committee',
                                      recommendation_label='Referral to committee',
-                                     css_class='default')
+                                     css_class='default',
+                                     merge_amendment_into_final=-1)
     state_2_9 = State.objects.create(name=ugettext_noop('needs review'),
```

Comment:
needs review should also be -1

Reply:
If the status is "needs review" and the recommendation is "Accepted", sure that it shouldn't be in the final version?

Reply:
If the status is need_review, it is not possible to jump to accepted because in this workflow needs_review is a final state (see next_states field). Needs review means something like adjourned here.

Reply:
Ok for me - after discussion with @CatoTH